lib/os.js

```diff
@@ -61,7 +82,4 @@ exports.getNetworkInterfaces = internalUtil.deprecate(function() {
 
 exports.EOL = isWindows ? '\r\n' : '\n';
 
-if (binding.isBigEndian)
-  exports.endianness = function() { return 'BE'; };
-else
-  exports.endianness = function() { return 'LE'; };
+binding.isBigEndian == true ? exports.endianness = function() { return 'BE'; } :  exports.endianness = function() { return 'LE'; };
```

Comment:
Ok that makes sense. I was just converted it to a ternary operator since it was like that to check if it the platform was windows on line 83 : https://github.com/amanuel2/node/blob/ddd82deab2f01384d7dea9272eaa320d77915442/lib/os.js#L83 , but this one is more chunky